src/app/core/services/external-cluster.ts

```diff
@@ -38,9 +38,9 @@ import {GCPDiskType, GCPMachineSize} from '@app/shared/entity/provider/gcp';
 @Injectable({providedIn: 'root'})
 export class ExternalClusterService {
   providerChanges = new BehaviorSubject<ExternalClusterProvider>(undefined);
```

Comment:
**Note for posterity reasons** 
This can't be changed into subject as Details page is dependent on preset or credentials so subscription is never activated. However it waits till provider changes but it never does keeping as BehaviorSubject will emit last saved provider value and will initialise component based on provider selected.